src/engine/Operation.cpp

```diff
@@ -65,7 +65,7 @@ shared_ptr<const ResultTable> Operation::getResult(bool isRoot,
 
   if (isRoot) {
     // Start with an estimated runtime info which will be updated as we go.
-    createRuntimeInfoFromEstimates();
+    createRuntimeInfoFromEstimates(std::make_shared<RuntimeInformation>());
```

Comment:
Looking at all the tests I had to adjust. Maybe most of them should either set isRoot, or if that's not an option, instead we should check for `_runtimeInformation || isRoot` here to prevent issues furhter down the line